docs/insomnia/custom-themes.md

```diff
@@ -5,7 +5,112 @@ category: "Plugins"
 category-url: plugins
 ---
 
-Make a custom Insomnia theme by creating an Insomnia [plugin](/insomnia/introduction-to-plugins/). To get started with some live examples, see our built-in [insomnia-plugin-themes](https://github.com/Kong/insomnia/tree/develop/plugins/insomnia-plugin-core-themes) module.
+Make a custom Insomnia theme by creating an Insomnia [plugin](/insomnia/introduction-to-plugins/).
+
+{:.alert .alert-primary}
+**Note**: To get started with some live examples, see our built-in [insomnia-plugin-themes](https://github.com/Kong/insomnia/tree/develop/plugins/insomnia-plugin-core-themes) module.
+
+## How to Build a Custom Theme
+
+1. Create a new project for your theme. If you plan to make this theme available via NPM, start your project name with `insomnia-plugin-`. The following example is called `insomnia-plugin-dark-colorblind-theme`.
+
+2. Following plugin instructions, write your [plugin package.json](https://docs.insomnia.rest/insomnia/introduction-to-plugins#plugin-packagejson). In your entry file, export your theme.
+
+3. Start with a baseline template. Each section of the configuration has `background` and `foreground` options, with seven (7) modifiable color options. 
+   
+   See the code comments for more information about which color option names correspond with which UI elements.
```

Comment:
I'm a bit torn on listing which property corresponds to which UI element, because it also depends on which `styles.*` property the colors are specified on.

The behavior with themes is more that users specify the bg/font colors for `surprise`, `danger`, `notice`, and the `highlight` colors of `xxs, xs, sm, ...` and Insomnia applies those to elements which could always change. This follows the bootstrap pattern to my knowledge.

Also maybe worth looking at VS Code and how they document [theme extensions](https://code.visualstudio.com/api/extension-guides/color-theme) though their [theme API](https://code.visualstudio.com/api/references/theme-color) is much better than ours. 

It's probably okay to give these example in comments but I think it's worth saying that it's not an exhaustive list?

Reply:
I can add that caveat. However, we need to give developers _some_ ground to stand on.

Reply:
Yep that's all good 👍🏽 thanks